packages/api/src/platforms/vtex/resolvers/skuVariations.ts

```diff
@@ -24,14 +24,14 @@ export const SkuVariants: Record<string, Resolver<Root>> = {
       root.isVariantOf.items,
       // Since `dominantVariantProperty` is a required argument, we can safely
       // access it.
```

Comment:
This comment is not true anymore 😅 
```suggestion
      // Since `dominantVariantProperty` is a required argument, we can safely
      // access it.
```